### DISCUSSION TITLE
关于d3d11-flip

### DISCUSSION THREAD

**Discussion by zhuceney** [hooke007/mpv_PlayKit#121] [mpv 通用]
发现设置"d3d11-flip=no"参数时在upscale2x视频（使用的是Anime4K_Upscale系列着色器）可以显著降低GPU占用，在我的3060上从原先的约80%降到50%。
禁用flip是否有其他副作用，没有的话也许可以设为默认值？

**Comment by zhuceney** [comment]
测试发现禁用flip导致显卡频率上升，整体功耗反而增加了😂

**Reply by yeezylife** [reply]
确实不能光看占用率！

**Comment by hooke007** [comment]
如果没有遇到bug就不要禁用它
理论上现代硬件上它的性能更好 https://docs.microsoft.com/en-us/windows/win32/direct3ddxgi/for-best-performance--use-dxgi-flip-model
